src/github3/pulls.py

```diff
@@ -926,6 +926,10 @@ class PullReview(models.GitHubCore):
 
         The SHA of the commit that the review was left on.
 
+        .. note::
+
+            It is possible for the attribute to be set to ``None``.
```

Comment:
Here is some suggested added words. I think this will wind up being the case.

```suggestion
            It is possible for the attribute to be set to ``None``, if the review references a commit
            that is no longer available in the pull request branch, such as after a force push.
```